app/controllers/rubygems/transfer/base_controller.rb

```diff
@@ -7,16 +7,14 @@ class Rubygems::Transfer::BaseController < ApplicationController
   rescue_from Pundit::NotAuthorizedError, with: :render_not_found
 
   def find_or_initialize_transfer
-    @rubygem = Rubygem.find_by(name: params[:rubygem_id])
     @rubygem_transfer = RubygemTransfer
       .includes(invites: :user)
       .where.not(status: :completed)
-      .find_or_initialize_by(created_by: Current.user, rubygem: @rubygem)
+      .find_or_initialize_by(created_by: Current.user)
```

Comment:
🤔 Do we have any constraints that would prevent `find_or_initialize_by` from fetching an unintentional record? i.e. Is it possible for there to be more than one `RubygemTransfer` created by the current user?